testsuite/tests/system/messages/test_automatic_emails.py

```diff
@@ -0,0 +1,70 @@
+"""
+Rewrite /spec/functional_specs/account_message_spec.rb
+When sending message to a developer account, the email is send
+"""
+import time
+
+import pytest
+
+from testsuite import rawobj
+from testsuite.utils import blame
+
+
+@pytest.fixture()
+def params(request):
```

Comment:
a fixture for an account is more appropriate than params. Unless account creation is part of test procedure, looking at the test it doesn't seem to be the case